database/database.go

```diff
@@ -67,27 +77,37 @@ func (db *Database) Config() *eth.Config {
 
 // ExecuteTx appends a transaction to the current block.
 func (db *Database) ExecuteTx(tx *ethTypes.Transaction) tmtAbciTypes.ResponseDeliverTx {
+	db.metrics.ExecutedTxsTotal.Add(1)
 	db.logger.Info("Executing DB TX", "hash", tx.Hash().Hex(), "nonce", tx.Nonce())
-
+	
+	txCost, _ := metrics.Web3FromWei(tx.Cost()).Float64()
+	db.metrics.TxsCostTotal.Add(txCost)
+	
+	txGasWei := new(big.Int).Mul(big.NewInt(int64(tx.Gas())), tx.GasPrice())
+	txGas, _ := metrics.Web3FromWei(txGasWei).Float64()
+	db.metrics.TxsGasTotal.Add(txGas)
```

Comment:
Althought ideally it would be great to have that metrics value available, this metric is useful and simple. We still need to work on a further iteration on the metrics we desire to track